.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         java: [ 11, 17 ]
-        bwc_version : [ "1.0.0", "1.1.0", "1.2.4", "1.3.2", "2.0.0", "2.1.0" ]
```

Comment:
Why aren't we bumping to 1.0.1 instead of just removing 1.0.0?

Reply:
I think the request is to use 1.1.0 as a lowest version https://amzn-aws.slack.com/archives/C02006YQC9X/p1659570733883399, removing makes sense.

Reply:
The fix that commit after 1.0.0 did not go into 1.0.1 release because 1.0.1 bundle is using 1.0.0 core + 1.0.0.0/1.0.1.0 plugins as a mix.